{
  "repository": {
    "name": "vert-x3/vertx-mysql-postgresql-client",
    "githubId": 27534295,
    "url": "https://github.com/vert-x3/vertx-mysql-postgresql-client",
    "isFork": false,
    "original": {
      "name": "",
      "githubId": 0,
      "url": ""
    },
    "isPullRequest": false,
    "pullRequestId": 0
  },
  "builds": {
    "buggyBuild": {
      "id": 418395625,
      "url": "http://travis-ci.org/vert-x3/vertx-mysql-postgresql-client/builds/418395625",
      "date": "Aug 20, 2018 10:11:57 PM"
    }
  },
  "reproductionBuggyBuild": {
    "reproductionDateBeginning": "Aug 20, 2018 10:13:34 PM",
    "reproductionDateEnd": "Aug 20, 2018 10:22:27 PM",
    "totalDuration": 529,
    "processDurations": {
      "cloning": {
        "nbSteps": 1,
        "totalDuration": 4,
        "stepNames": [
          "CloneRepository"
        ],
        "stepDurations": [
          4
        ]
      },
      "building": {
        "nbSteps": 2,
        "totalDuration": 41,
        "stepNames": [
          "CheckoutBuggyBuild",
          "BuildProject"
        ],
        "stepDurations": [
          0,
          41
        ]
      },
      "testing": {
        "nbSteps": 1,
        "totalDuration": 453,
        "stepNames": [
          "TestProject"
        ],
        "stepDurations": [
          453
        ]
      }
    },
    "projectRootPomPath": "./workspace/vert-x3/vertx-mysql-postgresql-client/418395625/pom.xml"
  },
  "patchDiff": {
    "files": {
      "numberAdded": 0,
      "numberChanged": 0,
      "numberDeleted": 0
    },
    "lines": {
      "numberAdded": 0,
      "numberDeleted": 0
    }
  },
  "projectMetrics": {
    "numberModules": 1,
    "numberSourceFiles": 22,
    "numberTestFiles": 16,
    "numberLibrariesFailingModule": 47
  },
  "tests": {
    "failingModule": "/root/workspace/vert-x3/vertx-mysql-postgresql-client/418395625",
    "overallMetrics": {
      "numberRunning": 51,
      "numberPassing": 13,
      "numberFailing": 2,
      "numberErroring": 36,
      "numberSkipping": 4,
      "failures": [
        {
          "failureName": "java.lang.AssertionError",
          "isError": false,
          "occurrences": 2
        },
        {
          "failureName": "java.util.concurrent.TimeoutException",
          "isError": true,
          "occurrences": 2
        },
        {
          "failureName": "io.netty.channel.AbstractChannel$AnnotatedConnectException",
          "isError": true,
          "occurrences": 34
        }
      ]
    },
    "failingClasses": [
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLClientTest",
        "numberRunning": 29,
        "numberPassing": 2,
        "numberFailing": 0,
        "numberErroring": 27,
        "numberSkipping": 0
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLTest",
        "numberRunning": 5,
        "numberPassing": 0,
        "numberFailing": 0,
        "numberErroring": 5,
        "numberSkipping": 0
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLConfigurationTest",
        "numberRunning": 2,
        "numberPassing": 0,
        "numberFailing": 0,
        "numberErroring": 2,
        "numberSkipping": 1
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLSslConfigurationTest",
        "numberRunning": 5,
        "numberPassing": 1,
        "numberFailing": 2,
        "numberErroring": 2,
        "numberSkipping": 1
      }
    ],
    "failureDetails": [
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLClientTest",
        "testMethod": "testByteA3Fields",
        "failureName": "io.netty.channel.AbstractChannel$AnnotatedConnectException",
        "detail": "Connection refused: localhost/127.0.0.1:5432",
        "isError": true
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLClientTest",
        "testMethod": "testUnhandledExceptionInHandlerRowStream",
        "failureName": "io.netty.channel.AbstractChannel$AnnotatedConnectException",
        "detail": "Connection refused: localhost/127.0.0.1:5432",
        "isError": true
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLClientTest",
        "testMethod": "testColumnsFromStream",
        "failureName": "io.netty.channel.AbstractChannel$AnnotatedConnectException",
        "detail": "Connection refused: localhost/127.0.0.1:5432",
        "isError": true
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLClientTest",
        "testMethod": "testUsingUUIDsInTables",
        "failureName": "io.netty.channel.AbstractChannel$AnnotatedConnectException",
        "detail": "Connection refused: localhost/127.0.0.1:5432",
        "isError": true
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLClientTest",
        "testMethod": "testByteA2Fields",
        "failureName": "io.netty.channel.AbstractChannel$AnnotatedConnectException",
        "detail": "Connection refused: localhost/127.0.0.1:5432",
        "isError": true
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLClientTest",
        "testMethod": "testInsertion",
        "failureName": "io.netty.channel.AbstractChannel$AnnotatedConnectException",
        "detail": "Connection refused: localhost/127.0.0.1:5432",
        "isError": true
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLClientTest",
        "testMethod": "testInsertedIds",
        "failureName": "io.netty.channel.AbstractChannel$AnnotatedConnectException",
        "detail": "Connection refused: localhost/127.0.0.1:5432",
        "isError": true
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLConfigurationTest",
        "testMethod": "testCharset",
        "failureName": "io.netty.channel.AbstractChannel$AnnotatedConnectException",
        "detail": "Connection refused: localhost/127.0.0.1:5432",
        "isError": true
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLClientTest",
        "testMethod": "testInstant",
        "failureName": "io.netty.channel.AbstractChannel$AnnotatedConnectException",
        "detail": "Connection refused: localhost/127.0.0.1:5432",
        "isError": true
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLSslConfigurationTest",
        "testMethod": "testQueryTimeout",
        "failureName": "java.util.concurrent.TimeoutException",
        "isError": true
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLConfigurationTest",
        "testMethod": "testQueryTimeout",
        "failureName": "java.util.concurrent.TimeoutException",
        "isError": true
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLClientTest",
        "testMethod": "testLongStream",
        "failureName": "io.netty.channel.AbstractChannel$AnnotatedConnectException",
        "detail": "Connection refused: localhost/127.0.0.1:5432",
        "isError": true
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLTest",
        "testMethod": "queryTypeTimestampWithoutTimezoneTest",
        "failureName": "io.netty.channel.AbstractChannel$AnnotatedConnectException",
        "detail": "Connection refused: localhost/127.0.0.1:5432",
        "isError": true
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLClientTest",
        "testMethod": "testSimpleConnection",
        "failureName": "io.netty.channel.AbstractChannel$AnnotatedConnectException",
        "detail": "Connection refused: localhost/127.0.0.1:5432",
        "isError": true
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLClientTest",
        "testMethod": "testSimpleSelect",
        "failureName": "io.netty.channel.AbstractChannel$AnnotatedConnectException",
        "detail": "Connection refused: localhost/127.0.0.1:5432",
        "isError": true
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLClientTest",
        "testMethod": "testRollingBackWhenNotInTransaction",
        "failureName": "io.netty.channel.AbstractChannel$AnnotatedConnectException",
        "detail": "Connection refused: localhost/127.0.0.1:5432",
        "isError": true
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLClientTest",
        "testMethod": "testUnhandledExceptionInHandlerResultSet",
        "failureName": "io.netty.channel.AbstractChannel$AnnotatedConnectException",
        "detail": "Connection refused: localhost/127.0.0.1:5432",
        "isError": true
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLSslConfigurationTest",
        "testMethod": "testPreferSslConfiguration",
        "failureName": "java.lang.AssertionError",
        "detail": "Expected true",
        "isError": false
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLClientTest",
        "testMethod": "testDateValueSelection",
        "failureName": "io.netty.channel.AbstractChannel$AnnotatedConnectException",
        "detail": "Connection refused: localhost/127.0.0.1:5432",
        "isError": true
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLClientTest",
        "testMethod": "testMultipleConnections",
        "failureName": "io.netty.channel.AbstractChannel$AnnotatedConnectException",
        "detail": "Connection refused: localhost/127.0.0.1:5432",
        "isError": true
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLClientTest",
        "testMethod": "testCommitWhenNotInTransaction",
        "failureName": "io.netty.channel.AbstractChannel$AnnotatedConnectException",
        "detail": "Connection refused: localhost/127.0.0.1:5432",
        "isError": true
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLTest",
        "testMethod": "queryTypeTimestampWithTimezoneTest",
        "failureName": "io.netty.channel.AbstractChannel$AnnotatedConnectException",
        "detail": "Connection refused: localhost/127.0.0.1:5432",
        "isError": true
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLClientTest",
        "testMethod": "testSelectionOfNullValues",
        "failureName": "io.netty.channel.AbstractChannel$AnnotatedConnectException",
        "detail": "Connection refused: localhost/127.0.0.1:5432",
        "isError": true
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLClientTest",
        "testMethod": "testByteA1Fields",
        "failureName": "io.netty.channel.AbstractChannel$AnnotatedConnectException",
        "detail": "Connection refused: localhost/127.0.0.1:5432",
        "isError": true
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLClientTest",
        "testMethod": "testSimpleStream",
        "failureName": "io.netty.channel.AbstractChannel$AnnotatedConnectException",
        "detail": "Connection refused: localhost/127.0.0.1:5432",
        "isError": true
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLClientTest",
        "testMethod": "testDecimalFields",
        "failureName": "io.netty.channel.AbstractChannel$AnnotatedConnectException",
        "detail": "Connection refused: localhost/127.0.0.1:5432",
        "isError": true
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLClientTest",
        "testMethod": "testSetAutocommitWhileInTransaction",
        "failureName": "io.netty.channel.AbstractChannel$AnnotatedConnectException",
        "detail": "Connection refused: localhost/127.0.0.1:5432",
        "isError": true
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLSslConfigurationTest",
        "testMethod": "testCharset",
        "failureName": "io.netty.channel.AbstractChannel$AnnotatedConnectException",
        "detail": "Connection refused: localhost/127.0.0.1:5432",
        "isError": true
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLClientTest",
        "testMethod": "testTimeColumn",
        "failureName": "io.netty.channel.AbstractChannel$AnnotatedConnectException",
        "detail": "Connection refused: localhost/127.0.0.1:5432",
        "isError": true
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLTest",
        "testMethod": "testUpdatingNumericField",
        "failureName": "io.netty.channel.AbstractChannel$AnnotatedConnectException",
        "detail": "Connection refused: localhost/127.0.0.1:5432",
        "isError": true
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLClientTest",
        "testMethod": "testUpdatingRow",
        "failureName": "io.netty.channel.AbstractChannel$AnnotatedConnectException",
        "detail": "Connection refused: localhost/127.0.0.1:5432",
        "isError": true
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLTest",
        "testMethod": "queryArrayTypeTest",
        "failureName": "io.netty.channel.AbstractChannel$AnnotatedConnectException",
        "detail": "Connection refused: localhost/127.0.0.1:5432",
        "isError": true
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLTest",
        "testMethod": "someTest",
        "failureName": "io.netty.channel.AbstractChannel$AnnotatedConnectException",
        "detail": "Connection refused: localhost/127.0.0.1:5432",
        "isError": true
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLClientTest",
        "testMethod": "testUnhandledExceptionInHandlerUpdateResult",
        "failureName": "io.netty.channel.AbstractChannel$AnnotatedConnectException",
        "detail": "Connection refused: localhost/127.0.0.1:5432",
        "isError": true
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLClientTest",
        "testMethod": "testRollback",
        "failureName": "io.netty.channel.AbstractChannel$AnnotatedConnectException",
        "detail": "Connection refused: localhost/127.0.0.1:5432",
        "isError": true
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLSslConfigurationTest",
        "testMethod": "testCorrectSslConfiguration",
        "failureName": "java.lang.AssertionError",
        "detail": "Expected true",
        "isError": false
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLClientTest",
        "testMethod": "testTwoTransactionsAfterEachOther",
        "failureName": "io.netty.channel.AbstractChannel$AnnotatedConnectException",
        "detail": "Connection refused: localhost/127.0.0.1:5432",
        "isError": true
      },
      {
        "testClass": "io.vertx.ext.asyncsql.PostgreSQLClientTest",
        "testMethod": "testInvalidInsertStatement",
        "failureName": "io.netty.channel.AbstractChannel$AnnotatedConnectException",
        "detail": "Connection refused: localhost/127.0.0.1:5432",
        "isError": true
      }
    ]
  },
  "commits": {
    "buggyBuild": {
      "repoName": "vert-x3/vertx-mysql-postgresql-client",
      "branchName": "master",
      "sha": "3481c6a92cc050511627176442102fc0abb04aee",
      "url": "http://github.com/vert-x3/vertx-mysql-postgresql-client/commit/3481c6a92cc050511627176442102fc0abb04aee",
      "date": "Aug 20, 2018 10:05:16 PM"
    }
  },
  "type": "only_fail"
}